linera-execution/src/wasm/conversions_from_wit.rs

```diff
@@ -47,7 +47,10 @@ impl From<contract::OutgoingMessage> for RawOutgoingMessage<Vec<u8>> {
         Self {
             destination: message.destination.into(),
             authenticated: message.authenticated,
-            is_skippable: message.is_skippable,
+            attributes: MessageAttributes {
+                is_skippable: message.is_skippable,
+                is_bouncing: false,
```

Comment:
(Is this because a message returned from an application must be a new one? Because bouncing is an explicit action?)

Reply:
yes

Reply:
also, I'm still tempted to hide `is_skippable` and make it a function of the destination (channel or direct)